assets/js/state/clusters.test.js

```diff
@@ -0,0 +1,19 @@
+import clustersReducer, { removeCluster } from '@state/clusters';
+import { clusterFactory } from '@lib/test-utils/factories';
+
+describe('Clusters reducer', () => {
+  it('should remove cluster from state', () => {
+    const cluster = clusterFactory.build();
+    const initialState = {
+      clusters: [cluster],
+    };
+
+    const action = removeCluster(cluster);
+
+    const expectedState = {
+      clusters: [],
```

Comment:
I still think that this test is weak. I would prefer having a removal from a list where you have more than one element